src/components/TopBar/TopBar.vue

```diff
@@ -46,6 +46,23 @@
 			</div>
 		</a>
 
+		<a v-if="showUpcomingEvent"
+			class="upcoming-event__card"
+			:aria-label="t('spreed', 'Next call')"
+			:href="nextEvent.calendarAppUrl"
+			:title="t('spreed', 'Click to open Calendar')"
+			target="_blank ">
+			<div class="icon">
+				<CalendarBlank :size="20" />
+			</div>
+			<div class="event-info">
+				<p class="event-info__header">
+					{{ t('spreed', 'Next call') }}
```

Comment:
should it say next meeting or call?
Could be an arranged chat session as well? cant really judge from the event?

Reply:
Technically, all of them are a call but the topic differs ( meeting, session...)

Reply:
I think, this was about the cases people are not really initiating a video/audio call.

But that really depends on the definition of `call`:
- an order or request for someone to be present 📆 
- an instance of speaking to someone on the phone 🤙🏽 
- a brief visit 👀 

I find `Next call` appropriate, but we can also consider `Next event`, `Next meeting`, or leave the note to translators what this is about, to represent correctly on another languages

Reply:
"Next call" is nice cause the button next to it says "Start call" so the language is the same.
But e.g. if we have a Talk conversation for the conference and the conference event is linked to that, it’s just "Next event". So that would be more generically applicable, to cover all sorts of cases.

Reply:
Knowing f a conversation was made for a conference or conference-like event is a bit tricky from info given in hand. Yet, We can use "Next event" if :
- Conversation is a public conversation 
OR
- Conversation has more than 50 participants (cannot be a team room)
OR
- the meeting is full day (cannot be a meeting or a call)